build.gradle

```diff
@@ -72,7 +72,7 @@ ext.deps = [
 
 allprojects {
   group = "com.linkedin.tony"
-  project.version = "0.2.0"
+  project.version = "0.3.0-SNAPSHOT"
```

Comment:
Is the SNAPSHOT necessary?

Reply:
This is temporary, will remove it, personal preference, don't want to mess up with maven central versions.